lib/chef/application.rb

```diff
@@ -151,6 +152,20 @@ def configure_logging
       Chef::Application.fatal!("Aborting due to invalid 'log_location' configuration", 2)
     end
 
+    # Turn `log_location :syslog` and `log_location :win_evt` into the
+    # appropriate loggers.
+    def configure_log_location
+      log_location = Chef::Config[:log_location]
+      return unless log_location.respond_to?(:to_sym)
```

Comment:
Doesn't this represent some kind of invalid configuration that we should explode on, so the user isn't surprised?